README.md

```diff
@@ -86,7 +86,8 @@ SwiftNIO            | Minimum Swift Version
 `2.43.0 ..< 2.51.0` | 5.5.2
 `2.51.0 ..< 2.60.0` | 5.6
 `2.60.0 ..< 2.65.0` | 5.7
-`2.65.0 ...`        | 5.8
+`2.65.0 ..< 2.75.0  | 5.8
+`2.75.0 ...`        | 5.9
```

Comment:
You've got an off-by-one error here: 2.75.0 has shipped and supports 5.8. Dropping 5.8 is a minor so this should be 2.76.0.